-------------------------------------------------------------------------------
Test set: com.poo.act9poo.Actividad9pooApplicationTests
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 11.874 s <<< FAILURE! - in com.poo.act9poo.Actividad9pooApplicationTests
contextLoads  Time elapsed: 0.002 s  <<< ERROR!
java.lang.IllegalStateException: Failed to load ApplicationContext
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'entityManagerFactory' defined in class path resource [org/springframework/boot/autoconfigure/orm/jpa/HibernateJpaConfiguration.class]: Invocation of init method failed; nested exception is org.hibernate.AnnotationException: @OneToOne or @ManyToOne on com.poo.act9poo.entities.Inscription.student references an unknown entity: com.poo.act9poo.entities.Student
Caused by: org.hibernate.AnnotationException: @OneToOne or @ManyToOne on com.poo.act9poo.entities.Inscription.student references an unknown entity: com.poo.act9poo.entities.Student

